src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/UnsafeHolder.kt

```diff
@@ -8,7 +8,7 @@
  * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-package org.jetbrains.kotlinx.lincheck
+package org.jetbrains.kotlinx.lincheck.transformation
```

Comment:
Please check that there is only one `UnsafeHolder` in the codebase and move it to the outer package -- it is now used not only for transformation.

Reply:
Moved it into `util` package.